.golangci.yaml

```diff
@@ -5,7 +5,6 @@ linter-settings:
 linters:
```

Comment:
I don't think these changes belong in this PR?

Reply:
They are a clean up as the linters got deprecated.

```
  
  level=warning msg="[runner] The linter 'structcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused."
  level=warning msg="[runner] The linter 'varcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused."
  level=warning msg="[runner] The linter 'deadcode' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused."
  ```


Do you want me to take them out?

Reply:
Thanks for clarifying.